### DISCUSSION TITLE
Setting static IPv6 routes

### DISCUSSION THREAD

**Discussion by jvfranklin** [rsmarples/dhcpcd#192] [General]
It's possible to use dhcpcd to configure static addresses and routes for IPv4, and static IPv6 addresses, but not a static IPv6 route. Is this something that could possibly be implemented?  We're adopting dhcpcd for use in our products, and fully static IPv6 is a use case we need to support.

Something like this configuration, by implementing a new "routers6" keyword, or retask "static routers" for both IPv4 and IPv6 possibly:
interface eth0
static ip6_address=2001:db8::2/64
static domain_name_servers=2001:db8::53 2001:db8::54
static routers6=fe80::1

If we could have it allow for a static route in addition to routes obtained via RA, that would be ideal.

**Comment by rsmarples** [comment]
The reasons for dhcpcd doing this for IPv4 are mainly for support of DHCP INFORM.
With IPv6, you could use your distro to add the static setup. Why use dhcpcd for this?
